### PR TITLE
migrate capi digitalocean jobs to community cluster

### PR DESCRIPTION
**Pull request by upodroid**
Creds were created in https://github.com/kubernetes/k8s.io/pull/6967

### CODE REVIEW

File: docs/job-migration-todo.md [hunk 1]
```diff
@@ -345,14 +325,7 @@ For more context, see the announcement thread at https://groups.google.com/a/kub
 |config/jobs/kubernetes/cloud-provider-vsphere/cloud-provider-vsphere-config.yaml|pull-cloud-provider-vsphere-e2e-test|[Search Results](https://cs.k8s.io/?q=name%3A%20pull-cloud-provider-vsphere-e2e-test%24&i=nope&files=&excludeFiles=&repos=)|
 |config/jobs/kubernetes/cloud-provider-vsphere/cloud-provider-vsphere-config.yaml|pull-cloud-provider-vsphere-e2e-test-on-latest-k8s-version|[Search Results](https://cs.k8s.io/?q=name%3A%20pull-cloud-provider-vsphere-e2e-test-on-latest-k8s-version%24&i=nope&files=&excludeFiles=&repos=)|
 |config/jobs/kubernetes/kops/kops-periodics-misc.yaml|ci-kops-build|[Search Results](https://cs.k8s.io/?q=name%3A%20ci-kops-build%24&i=nope&files=&excludeFiles=&repos=)|
-|config/jobs/kubernetes/kops/kops-periodics-misc.yaml|e2e-kops-do-calico-dns-none|[Search Results](https://cs.k8s.io/?q=name%3A%20e2e-kops-do-calico-dns-none%24&i=nope&files=&excludeFiles=&repos=)|
```
Comment (upodroid) [inline docs/job-migration-todo.md:348]: Moved in https://github.com/kubernetes/test-infra/pull/32189

**Comment by BenTheElder** [comment]
A lot of these are lacking resource requests.

I'd suggest moving one sample job and working through the issues there first?

**Pull request merged**